### PR TITLE
Replace get a quote with a contact form using contact-form-api on the backend

### PR DESCRIPTION
**Pull request by hypesystem**
This shows off some of the work we do :smile: @jacobhschriver better be happy 😉 

image

image

### CODE REVIEW

File: _includes/get_a_quote.html [hunk 1]
```diff
@@ -8,180 +8,287 @@ <h1>Get a quote</h1>
         We are experts in working in the most efficient way possible, playing to each others' strengths, making sure you get the most value delivered for your money.
     </p>
     <p>
-        When you hire us, we will have a trial period of a month, after which we will evaluate our working relationship, to ensure that you are happy with the work we are doing for you.
+        If you get in touch, we will first arrange a meeting to talk about your concrete business situation.
```
Comment (HelenaBach) [inline _includes/get_a_quote.html]: I think that we should ship the 'if' part and make the sentence more direct and 'active'.
Something like:

Get in touch, and get a (free) meeting to talk about your concrete business situation.

File: _includes/get_a_quote.html [hunk 2]
```diff
@@ -8,180 +8,287 @@ <h1>Get a quote</h1>
...
+        We will suggest a small project for you to get familiar with how we work, and then hopefully initiate a longer, sustainable relationship, where we help you improve your business's software.
```
Comment (HelenaBach) [inline _includes/get_a_quote.html]: 'We usually suggest a small project..'
I think that 'and then hopefully' sound a bit like we don't believe in our self. Maybe we could expand it to something like:
'.. and if we're a good match then hopefully..' / 'if the fit is right then hopefully..'

File: _includes/get_a_quote.html [hunk 3]
```diff
@@ -8,180 +8,287 @@ <h1>Get a quote</h1>
...
     </p>
     <p>
-        If you want to hire us, see if we can make something happen within your budget here:
+        Tell us a bit about your business and why you are considering getting help building software. Get in touch if you want to hear from us:
```
Comment (HelenaBach) [inline _includes/get_a_quote.html]: I think the form is quite self-explanatory and I think that the text risk sounding like we require that the potential costumer needs to be too clarified about what they want. So maybe we don't need this paragraph at all, or maybe make it shorter and more open:

If you got any questions, got an idea or a problem or simply want to get in touch, then don't hesitate to contact us:

File: _includes/get_a_quote.html [hunk 4]
```diff
@@ -8,180 +8,287 @@ <h1>Get a quote</h1>
...
+                    <input type="text" name="subject" value="Quote request" placeholder="Quote request">
+                </label>
+                <label for="message">
+                    <span>Message</span>
+                    <textarea name="message" placeholder="Hi deranged,
 
-        (function() {
-            let button = document.querySelector("[data-tab=team] .button");
-            let evalText  = document.querySelector("[data-tab=team] .evaluation-text");
+We're planning to expand into a new market, but we need our technology to keep up..."></textarea>
```
Comment (HelenaBach) [inline _includes/get_a_quote.html:35]: really nice placeholder text :)

File: _includes/get_a_quote.html [hunk 5]
```diff
@@ -8,180 +8,287 @@ <h1>Get a quote</h1>
...
+                    });
                 }
-                if(val < 50000) {
-                    return setEvalText("bad", `It's pretty hard to run a project for less that 50.000 DKK&mdash;but you can get in touch and we can talk through a realistic budget.`);
+
+                function showFailure() {
+                    contactFormOverlay.classList.remove("contact-form-overlay-visible");
+                    
+                    setTimeout(function() {
+                        contactFormOverlay.innerText = "Something went wrong - try again!";
```
Comment (HelenaBach) [inline _includes/get_a_quote.html:129]: Might want to tell the user if it's one of the fields that are invalid (fx typing invalid email).

**Review by HelenaBach — COMMENTED** [review]
Really nice form!

I cant find the get_a_quote.scss file, but I think that it looks better and the page look more coherent if the paragraphs have a max width of around 800px, instead of 500px.

File: _includes/get_a_quote.html (hunk 5 above)
Comment (hypesystem) [reply _includes/get_a_quote.html:129]: v2 :smile:

**Comment by hypesystem** [comment]
Fixes made @HelenaBach -- thoughts?

File: _includes/get_a_quote.html [hunk 6]
```diff
@@ -8,11 +8,8 @@ <h1>Get a quote</h1>
         We are experts in working in the most efficient way possible, playing to each others' strengths, making sure you get the most value delivered for your money.
     </p>
     <p>
-        If you get in touch, we will first arrange a meeting to talk about your concrete business situation.
-        We will suggest a small project for you to get familiar with how we work, and then hopefully initiate a longer, sustainable relationship, where we help you improve your business's software.
-    </p>
-    <p>
-        Tell us a bit about your business and why you are considering getting help building software. Get in touch if you want to hear from us:
+        Get in touch and we will first arrange a meeting to talk about your concrete business situation.
+        We will suggest a small project for you to get familiar with how we work, and if the fit is right initiate a longer, sustainable relationship, where we help you improve your business's software.
```
Comment (HelenaBach) [inline _includes/get_a_quote.html:12]: update to 'we usually suggest a small project'
This makes it sound more flexible

**Review by HelenaBach — APPROVED** [review]
Looks super nice! 

After revisiting the site, I actually think that we should consider to bumb up the text size of all the text having the smallest size, and make a bigger 'linjeafstand'. Nothing we need to do right away, but something we should deffo consider. 

This is to make it look lighter and less text dense and I think it would look a bit more smooth and '2019 site' ish.

**Comment by hypesystem** [comment]
@HelenaBach sounds like something we could defs try! If you wanna give it a go, feel free to do a PR :smile:

**Pull request merged**